docs/r2/tensorboard_quickstart.ipynb

```diff
@@ -0,0 +1,463 @@
+      ]
+    },
+    {
+      "metadata": {
+        "id": "j-DHsby18cot",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "mnist = tf.keras.datasets.mnist\n",
+        "\n",
+        "(x_train, y_train),(x_test, y_test) = mnist.load_data()\n",
+        "x_train, x_test = x_train / 255.0, x_test / 255.0\n",
+        "\n",
+        "def create_model():\n",
+        "  return tf.keras.models.Sequential([\n",
+        "    tf.keras.layers.Flatten(input_shape=(28, 28)),\n",
+        "    tf.keras.layers.Dense(512, activation='relu'),\n",
+        "    tf.keras.layers.Dropout(0.2),\n",
+        "    tf.keras.layers.Dense(10, activation='softmax')\n",
+        "  ])"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "XKUjdIoV87um",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "## Using TensorBoard with Keras Model.fit()"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "8CL_lxdn8-Sv",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "When training with Keras's [Model.fit()](https://www.tensorflow.org/api_docs/python/tf/keras/models/Model#fit), adding the [TensorBoard callback](https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/keras/callbacks/TensorBoard) will ensure logs are created and stored. Additionally, enable histogram computation every epoch with `histogram_freq=1` (this is off by default)\n",
+        "\n",
+        "Place the logs in a timestamped subdirectory to allow easy selection of different training runs."
+      ]
+    },
+    {
+      "metadata": {
+        "id": "WAQThq539CEJ",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "model = create_model()\n",
+        "model.compile(optimizer='adam',\n",
+        "              loss='sparse_categorical_crossentropy',\n",
+        "              metrics=['accuracy'])\n",
+        "\n",
+        "log_dir=\"logs/fit/\" + datetime.now().strftime(\"%Y%m%d-%H%M%S\")\n",
+        "tensorboard_callback = tf.keras.callbacks.TensorBoard(log_dir=log_dir, histogram_freq=1)\n",
+        "\n",
+        "model.fit(x=x_train, \n",
+        "          y=y_train, \n",
+        "          epochs=5, \n",
+        "          validation_data=(x_test, y_test), \n",
+        "          callbacks=[tensorboard_callback])"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "asjGpmD09dRl",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "Start TensorBoard through the command line or within a notebook experience. The two interfaces are generally the same. In notebooks, use the `%tensorboard` line magic. On the command line, run the same command without \"%\"."
```

Comment:
Is it clear enough that by “on the command line” we’re not referring to
`!tensorboard` in Colab (which will block, and also show no output)?

Reply:
I think so, as we say the alternative is within a notebook.